src/_pytest/cacheprovider.py

```diff
@@ -282,9 +282,13 @@ def pytest_addoption(parser):
     )
     group.addoption(
         "--cache-show",
-        action="store_true",
+        action="append",
+        nargs="?",
```

Comment:
There might be a better way?

Reply:
I've tried a few things (action="store" with default="*") but that didn't work, so I guess this is fine.

Reply:
Ok, borrowed / inspired by pytest-cov's `--cov`.